src/index.html

```diff
@@ -7,8 +7,105 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="logo">
+
+        <a href="home-page">
+
+          <img
+          class="logo_pic"
+          src="images/logo.png"
+          alt="MOYO_Logo">
+
+        </a>
+
+      </div>
+
+      <nav class="nav">
+
+        <ul class="list">
+
+          <li class="listItem listItem1">
+
+            <a href="apple" class="navItem navItem1 is-active">
```

Comment:
1. `ul`, `li`, `a` should be as elements of nav block (BEM)
2. no needs to use such classes as navItem1, navItem2 etc
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="apple" class="nav__link is-active">
```